platforms/android/library/src/main/java/io/element/android/wysiwyg/utils/HtmlConverter.kt

```diff
@@ -0,0 +1,18 @@
+package io.element.android.wysiwyg.utils
+
+import androidx.core.text.HtmlCompat
+
+internal interface HtmlConverter {
+    fun fromHtmlToPlainText(html: String): String
+}
+
+internal object AndroidHtmlConverter: HtmlConverter {
+    /**
+     * Get the content with formatting removed.
+     * TODO: Return markdown formatted plaintext instead
+     */
+    override fun fromHtmlToPlainText(html: String): String =
+        HtmlCompat.fromHtml(
```

Comment:
We could probably use `HtmlToSpansParser` here since it's our custom Html parser.

Reply:
I'm not sure the HTML to spans implementation necessarily needs to overlap with the HTML to plain text implementation as eventually converting HTML to plain text will be done completely separately (using the Rust model) anyway. I could see value in combining the two interfaces though. What are your thoughts?

Reply:
I've refactored a bit to combine the HTML utilities under one interface: [`feecd52`](https://github.com/matrix-org/matrix-wysiwyg/pull/235/commits/feecd5269d8943a89d3cc57f33bd0a1ef931a0e6).